File: Jumping Number (Special Numbers Series #4).py
def jumping_number(number):
    number = str(number)
    flag = True
    for i in range(1,len(number)):
        if abs(int(number[i])-int(number[i-1]))==1:
            continue
        else:
            flag = False
            break
    return "Jumping!!" if flag else "Not!!"

File: Jumping Number (Special Numbers Series #4)/Jumping Number (Special Numbers Series #4).js
function jumpingNumber(n){
  n = String(n)
  if(n.length==1)return "Jumping!!"
  return [...Array(String(n).length-1).keys()].map(i => n.slice(i,i+2)).every(i => "0123456789876543210".indexOf(i)!=-1) ? "Jumping!!" : "Not!!"
}
